README.md

```diff
@@ -168,7 +168,7 @@ All official images should provide a consistent interface. A beginning user shou
 3.	If the image only contains the main executable and its linked libraries (ie no shell) then it is fine to use the executable as the `ENTRYPOINT`, since that is the only thing that can run:
 
 	```Dockerfile
-	ENTRYPOINT ["swarm"]
+	ENTRYPOINT ["mybinary"]
```

Comment:
why??

Reply:
Thought it didn't make sense to use `swarm` as an example; this is just for illustration?

Reply:
I think we chose `swarm` since it was one of the only single binary images we had at the time. I guess `hello` from the hello-world image would work.

Reply:
It might be more clear if we choose something more self-descriptive like `fully-static-binary`